README.md

```diff
@@ -1,3 +1,271 @@
-# swiftplot
+<div><img src="swiftplot.png" width="600"></div>
+<br>
 
-A description of this package.
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+<br>
+
+# Table of contents
+
+  * [Overview](#overview)
+  * [How to include the library in your package](#how-to-include-the-library-in-your-package)
+  * [How to include the library in your Jupyter Notebook](#how-to-include-the-library-in-your-jupyter-notebook)
+  * [Examples](#examples)
+    * [Simple Line Graph](#simple-line-graph)
+    * [Line Graph with multiple series of data](#line-graph-with-multiple-series-of-data)
+    * [Line Graph with Sub Plots stacked horizontally](#line-graph-with-sub-plots-stacked-horizontally)
+    * [Plot functions using LineGraph](#plot-functions-using-linegraph)
+    * [Displaying plots in Jupyter Notebook](#displaying-plots-in-jupyter-notebook)
+  * [How does this work ?](#how-does-this-work)
+  * [Documentation](#documentation)
+  * [Limitations](#limitations)
+  * [Credits](#credits)
+  * [License](#license)
+
+## Overview
+The SwiftPlot framework is a cross-platform library that lets you plot graphs natively in Swift.
+The existing Swift plotting frameworks (such as CorePlot) run only on iOS or Mac.
+The idea behind SwiftPlot is to create a cross-platform library that runs on iOS, Mac, Linux and Windows.
+</br>
+</br>
+SwiftPlot currently uses two rendering backends to generate plots:
+- Anti-Grain Geometry(AGG) C++ rendering library
+- A simple SVG Renderer
+
+To encode the plots as PNG images it uses the [lodepng](https://github.com/lvandeve/lodepng) library.
+</br>
+SwiftPlot can also be used in Jupyter Notebooks.
+</br>
+Examples, demonstrating all the features, have been included with the repository under the `examples` directory.
+To run the examples, clone the repository, and run the run_examples.sh script as shown below.
+
+```console
+run_examples.sh
+```
+
+The resultant images are stored in the `examples/Reference` directory. The images rendered by each of the backends are stored their respective directories: agg and svg.Examples
```

Comment:
Is the Examples at the end here an extension to the directory (svg.Examples) or it is something that was left in there accidentally? If it is an extension, maybe it should be added to the agg case as well to avoid confusion.

Reply:
It was an accident. I'll correct it immediately